infra/apim_v2/apim.tf

```diff
@@ -378,7 +378,7 @@ module "apim_external_api_ms_v2" {
     },
     {
       operation_id = "getInstitutionsUsingGET"
-      xml_content = templatefile("./api/base_ms_url_external_policy.xml.tpl", {
+      xml_content = templatefile("./api/base_ms_url_external_product_policy.xml.tpl", {
```

Comment:
why this change?

Reply:
This is because external_product_policy enables setting a productId filter based on the subscription key used.